<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiRenameDialog</class>
 <widget class="QDialog" name="multiRenameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MultiRename</string>
  </property>
  <widget class="QComboBox" name="prefixComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>プレフィックス</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>a~z</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>first</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>I~X</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="newName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;new name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="suffixComboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>サーフィックス</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>a~z</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>first</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>I~X</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="renamePushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>180</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>名前変更</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
